cleanlab/object_detection/summary.py

```diff
@@ -34,6 +43,412 @@
+            else:
+                lab_label = (
+                    sample["lab_labels"][max_iou_idx]
+                    if class_names is None
+                    else class_names[str(sample["lab_labels"][max_iou_idx])]
+                )
+                if pred_label == lab_label:
+                    # TP case: the predicted bounding box overlaps with a ground truth bounding box with the same class label
+                    class_metrics[lab_label]["TP"] += 1
+                    matched_label_idx.add(max_iou_idx)
+                else:
+                    # FP case: the predicted bounding box overlaps with a ground truth bounding box with a different class label
+                    class_metrics[lab_label]["FP"] += 1
+
+        # FN case - any ground truth bounding box that does not have an overlap with any predicted bounding box
+        missed_labels = set(range(len(iou_t[0]))) - matched_label_idx
+        for idx in missed_labels:
+            lab_label = (
+                sample["lab_labels"][idx]
+                if class_names is None
+                else class_names[str(sample["lab_labels"][idx])]
+            )
+            class_metrics[lab_label]["FN"] += 1
+
+    for c in class_metrics.keys():
+        acc_score = class_metrics[c]["TP"] / (
+            class_metrics[c]["TP"] + class_metrics[c]["FP"] + class_metrics[c]["FN"]
+        )
+        class_metrics[c]["accuracy"] = round(acc_score, 2)
+
+    return class_metrics
+
+
+def get_sorted_bbox_count_idxs(labels, predictions):
+    """
+    Returns a tuple of idxs and bounding box counts of images sorted from highest to lowest number of bounding boxes.
+
+    This plot can help you discover images with abnormally many/few object annotations.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    predictions:
+        Predictions output by a trained object detection model.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+
+    Returns
+    -------
+    sorted_idxs: List[Tuple[int, int]], List[Tuple[int, int]]
+        A tuple containing two lists. The first is an array of shape ``(N,)`` containing the number of annotated objects for each image in the dataset.
+        The second is an array of shape ``(N,)`` containing the number of predicted objects for each image in the dataset.
+    """
+    lab_count, pred_count = object_counts_per_image(labels, predictions)
+    lab_grouped = zip([i for i in range(len(labels))], lab_count)
+    pred_grouped = zip([i for i in range(len(predictions))], pred_count)
+
+    sorted_lab = sorted(lab_grouped, key=lambda x: x[1], reverse=True)
```

Comment:
as far as I understood, you are trying to get indices after sorting. if that is correct, here is an alternative way:

```
sorted_indices = [index for index, _ in sorted(enumerate(original_list), key=lambda x: x[1], reverse=True)]
```

here is a better way is with numpy:

```
original_array = np.array([4, 2, 7, 1, 9])
sorted_indices = np.argsort(original_array)
```

Reply:
Numpy way is good although what is contributed here is also acceptable